packages/tokenlist/public/superfluid.tier-b.tokenlist.json

```diff
@@ -321,30 +321,6 @@
       "logoURI": "https://raw.githubusercontent.com/superfluid-finance/assets/master/public/tokens/op/icon.svg",
       "tags": ["underlying"]
     },
-    {
-      "address": "0x4caB5B9930210E2EdC6A905B9C75D615872A1A7E",
```

Comment:
I wonder why this one disappeared

Reply:
the only reason i can imagine is that it's not in subgraph, and was added manually.